ngraph/test/type_prop/power.cpp

```diff
@@ -0,0 +1,265 @@
+//*****************************************************************************
```

Comment:
I think it's better to generalize this test to avoid code duplication across all of elementwise ops like we did in [unary ops](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/type_prop/unary_ops.cpp). @ilyachur I was told you discouraged this approach, can you comment why?

Reply:
@jdanieck We can discuss it offline.
I prefer to have separate file for each operation (because it allows to demarcate different operations).
But I also dislike a code duplication. I think we can use parametrised tests in order to avoid code duplication.

Reply:
Ok, I'll call you tomorrow. Let's merge this one as is to not block Piotr with his task.  He will work on the next element wise op after this one so if we agree on some parameterized version he can implement it in the next PR.